### DISCUSSION TITLE
There is a bug about cpp head

### DISCUSSION THREAD

**Discussion by Jackqhr** [thebaselab/codeapp#1011] [Q&A]
I noticed developers add support of bits/stdc++.h
But in this head ,it include another head called setjmp.h.And there is no head called setjmp.h in the local clang compiler.
I have submitted this bug in testflight.
![image](https://github.com/thebaselab/codeapp/assets/43907194/b03b279c-9e45-4762-a7f6-154e5ddd45c9)
![image](https://github.com/thebaselab/codeapp/assets/43907194/5936eb18-5651-4ba5-91e7-9240a80bf1e9)

**Comment by bummoblizard** [comment]
Thank you for reporting. This should be fixed in the TestFlight build.
